operator/controllers/seldondeployment_engine.go

```diff
@@ -221,6 +221,11 @@ func createExecutorContainer(mlDep *machinelearningv1.SeldonDeployment, p *machi
 		}
 	}
 
+	probeScheme := corev1.URISchemeHTTP
+	if utils.IsEmptyTLS(p) {
+		probeScheme = corev1.URISchemeHTTPS
+	}
```

Comment:
Should this be `if !utils.IsEmptyTLS(p)`?

Reply:
Good catch, thanks!